article/test/ArticleFeatureTest.scala

```diff
@@ -494,7 +494,6 @@ import collection.JavaConversions._
         findFirst("aside").getAttribute("role") should be("complementary")
         findFirst("header").getAttribute("role") should be("banner")
         findFirst(".l-footer__secondary").getAttribute("role") should be("contentinfo")
-        findFirst("nav").getAttribute("role") should be("navigation")
```

Comment:
Because `<nav>` has the role baked into it's semantics. There is no meaning in duplicating the logic as an `aria-role` except for browsers who would treat `nav` as `div`. I think we don't support any of them anymore.